src/solver/postprocess.jl

```diff
@@ -105,6 +104,32 @@ function _postprocess_root(x::SymbolicUtils.BasicSymbolic)
         end
     end
 
+    arg = arguments(x)[1]
+    if oper === acos
+        if arg === 0
+            return Symbolics.term(/, pi, 2)
+        elseif arg === 1
+            return 0
+        end
+    elseif oper === asin
+        if arg === 0
+            return 0
+        elseif arg === 1
+            return Symbolics.term(/, pi, 2)
+        end
+    end
+
+    if oper === (+)
+        args = arguments(x)
+        for arg in args
+            if isequal(arg, 0)
+                after_removing = setdiff(args, arg)
+                isone(length(after_removing)) && return after_removing[1]
+                return Symbolics.term(+, after_removing)
+            end
+        end
+    end
+
```

Comment:
Seems like we should define a more general rule set for this?

Reply:
ah right my bad, will fix